lib/game_teams_manager.rb

```diff
@@ -117,4 +117,17 @@ def goals_by_team_id
     end
     goals_by_team_id
   end
+
+  def team_accuracy(game_ids)
+    # selects all of the games in a season
+    games = @game_teams.select {|game_team| game_ids.include?(game_team.game_id)}
+    #groups them by team
+    games_by_team = games.group_by {|game| game.team_id}
+    # update values to accuracy
+    team_accuracy = games_by_team.transform_values do |game_teams|
```

Comment:
this method combines the filtering by team, and the calculating of the accuracy. I bet we could have a single accuracy method to operate on any collection of game_team objects.